### PR TITLE
[On-boarding] Collect screen data to improve future on-boarding.

### PR DESCRIPTION
**Pull request by alloy**
Nothing fancy, just stores entities that have slugs. How/if we’re going to use this data in the future is not yet defined.

NSA

### CODE REVIEW

File: Artsy/App/ARAnalyticsPapertrail.m [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        if (slugs.count > 0) {
+            NSError *error = nil;
+            [NSJSONSerialization writeJSONObject:@{ @"screen": pageTitle, @"slugs": slugs }
+                                        toStream:self.stream
+                                         options:0
+                                           error:&error];
+            NSAssert(error == nil, @"Unable to save analytics papertrail: %@", error);
+            NSInteger written = [self.stream write:"\n" maxLength:1];
+            NSAssert(written == 1, @"Unable to append newline.");
+        }
```
Comment (orta) [inline Artsy/App/ARAnalyticsPapertrail.m]: got any opinions on `NSOutputStream` + `NSJSONSerialization` for battery life?

Comment (alloy) [inline Artsy/App/ARAnalyticsPapertrail.m]: In what way?

Comment (orta) [inline Artsy/App/ARAnalyticsPapertrail.m]: I almost wrote "Could the always open stream not let the hard drive go to sleep".

Comment (alloy) [inline Artsy/App/ARAnalyticsPapertrail.m]: Owww, hah, nah, iOS can handle open devices, there will be many at any given time.

**Comment by orta** [comment]
Cool, shippit.

**Pull request merged**